pywsitest/utils.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+
+PATH_REGEX = re.compile(r"^\$\{(.*)\}$")
+
+
+def get_resolved_value(response: dict, path: str) -> object:
+    """
+    Retrieves a value from a dictionary at a given path
+
+    Parameters:
+        response (dict): The response to check against for a value
```

Comment:
Any reason why path is not one of the parameters in the docstring?

Reply:
It was copied across when I refactored, and I forgot to add it
Thanks for spotting this, fixed in 86612d2